torch/csrc/distributed/c10d/FakeProcessGroup.hpp

```diff
@@ -20,7 +20,21 @@ class FakeWork : public Work {
 
 class FakeProcessGroup : public Backend {
  public:
-  FakeProcessGroup(int rank, int size) : Backend(rank, size) {}
+  struct Options : Backend::Options {
+    explicit Options() : Backend::Options("fake") {}
+
+    int fake_option = 0;
+  };
```

Comment:
Needed to define options for the fake PG in order to test the options-only override

Reply:
i guess this is fine. Maybe later we can replace it with a real option